src/azure-cli/azure/cli/command_modules/storage/_breaking_change.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.breaking_change import register_other_breaking_change
+
+register_other_breaking_change('storage account migration start',
+                               message='Starting from 5/19, changing redundancy configuration would require additional '
```

Comment:
The release date could be changed. Better to use future version instead of future date